### DISCUSSION TITLE
Add Libre.fm or ListenBrainz scrobbling function.

### DISCUSSION THREAD

**Discussion by OctopusET** [ciderapp/Cider#819] [Feature Request]
For now, Cider only supports Last.fm scrobbling. I can sync Last.fm with Libre.fm or ListemBrainz. But it would be much better if there are options for them.

I guess this could be reference.
https://github.com/web-scrobbler/web-scrobbler

Thanks.

**Comment by cryptofyre** [comment]
Node modules would be more preferable, but if you did want it enough you could always make a Cider plugin.

**Comment by hcuk94** [comment]
Came here to suggest the same, but also wanted to add that both ListenBrainz and Libre.fm APIs are swap-out replacements for Last.fm, so it should be fairly straightforward to add them as options - a change to the API URL plus an auth flow for each.

**Comment by kalmenn** [comment]
All in on ListenBrainz integration as Cider is currently the only app that is preventing me from switching from last.fm

**Comment by kgarner7** [comment]
I've made a plugin for this (currently just ListenBrainz) https://github.com/kgarner7/cider-music-listenbrainz-plugin

**Reply by wsyxbcl** [reply]
Great work, thank you!
